integration_tests/test-environment-async/test-environment.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const JSDOMEnvironment = require('../../packages/jest-environment-jsdom');
+
+class TestEnvironment extends JSDOMEnvironment {
+  constructor(config) {
+    super(config);
+  }
+
+  setup() {
+    return super.setup().then(() => {
+      this.global.setup = 'setup';
+    });
+  }
+
+  teardown() {
+    return super.teardown().then(() => {
+      mkdirp.sync('/tmp/jest');
```

Comment:
should this go through https://nodejs.org/api/os.html#os_os_tmpdir? Will that allow us to run the test on windows?

Reply:
had to update rimraf to make it work